### PR TITLE
Broaden scribd regex to include `document` format urls.

### PR DESCRIPTION
**Pull request by ryanscottaudio**
Addresses https://app.asana.com/0/315489207434239/319041693664168.

### CODE REVIEW

**Review by danmakenoise — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -35,14 +35,15 @@
...
     "browserify": "^13.0.0",
+    "devtool": "^2.3.1",
     "faucet": "0.0.1",
     "hyperscript": "^1.4.7",
     "nyc": "^8.3.0",
     "query-dom": "^3.0.7",
     "semistandard-deku": "micnews/semistandard#deku",
     "snazzy": "^5.0.0",
+    "tapava": "^2.3.0",
     "tape-catch": "^1.0.4",
```
Comment (ellell) [inline package.json]: @ryanscottaudio are we using both tapava and tape-catch? Or can tape-catch be removed?

Comment (ryanscottaudio) [reply package.json]: p sure this is outdated; `tapava` isn't in the most up-to-date diff at all

Comment (ellell) [reply package.json]: @ryanscottaudio what do you mean?

Comment (ryanscottaudio) [reply package.json]: tapava isnt in https://github.com/micnews/embeds/pull/87/files

Comment (ellell) [reply package.json]: ooh, sorry totally missed that.

**Pull request merged**